pkg/lib/bundle/testdata/csvless_bundle/prometheus_v0.22.2/metadata/dependency_file.yaml

```diff
@@ -0,0 +1,13 @@
+dependencies:
```

Comment:
These 3 dependencies are identical. Is there a reason for this? I would prefer 1 olm.gvk and 1 olm.package type. That should be sufficient.

Reply:
I still need 2 to see how it handles unique gvk. but yeah changed !